dashboard/app/controllers/api/v1/pd/workshop_survey_report_controller.rb

```diff
@@ -106,5 +100,11 @@ def local_workshop_survey_report
         end
       end
     end
+
+    private
+
+    def facilitator_name_filter
+      current_user.facilitator? && !current_user.workshop_organizer? ? current_user.name : nil
+    end
```

Comment:
I think I need to rephrase this part some.

If a user is a facilitator but not a workshop organizer, then when we are doing averaging of workshops, we need to only look at responses for that specific facilitator for facilitator-specific questions. This means we need to know the facilitator's name to do filtering.

If the user is a facilitator and  workshop organizer (like with CSF workshops), then we should average results for all facilitators, but refrain from having answers broken out for the all_my_* answers.

let me rewrite this a bit